lib/bolt/pal.rb

```diff
@@ -52,12 +52,22 @@ def initialize(modulepath, hiera_config, resource_types, max_compiles = Etc.npro
       @apply_settings = apply_settings
       @max_compiles = max_compiles
       @resource_types = resource_types
+      @project = project
 
       @logger = Logging.logger[self]
       if modulepath && !modulepath.empty?
         @logger.info("Loading modules from #{@modulepath.join(File::PATH_SEPARATOR)}")
       end
 
+      # If the project name is the same as one of the builtin modules raise a warning
+      bolt_mods = Dir.entries(BOLTLIB_PATH)
```

Comment:
```suggestion
      bolt_mods = Dir.children(BOLTLIB_PATH)
```

`entries` includes `.` and `..` while `children` does not.